Gemfile

```diff
@@ -1,3 +1,5 @@
 source 'https://rubygems.org'
 
-gem 'graphql_swift_gen', path: 'Dependencies/Swift Gen'
\ No newline at end of file
+git 'https://github.com/Shopify/graphql_swift_gen', branch: 'task/buy-sdk-3' do
```

Comment:
I'm guessing you're branching off of develop and into master, but maybe should  branch off master in this case. I opened a PR to take care of these commits.

Also I notice you re-based so the Gemfile changes and the build script changes are in one commit, I'd prefer to keep them separate since they were made for different reasons (one fixes a problem with our external dependency management, and one is fixing a scripting problem).

Reply:
Done